lib/vanagon/patch.rb

```diff
@@ -32,7 +32,7 @@ def initialize(path, strip, fuzz, after, destination)
     end
 
     def cmd(platform)
-      "#{platform.patch} --strip=#{@strip} --fuzz=#{@fuzz} --ignore-whitespace --no-backup-if-mismatch < $(workdir)/patches/#{File.basename(@path)}"
+      "#{platform.patch} --strip=#{@strip} --fuzz=#{@fuzz} --ignore-whitespace --no-backup-if-mismatch < ${workdir}/patches/#{File.basename(@path)}"
```

Comment:
I'm going to run an ad-hoc build of the agent with this change to make sure it doesn't break anything. Note that if it does, then dmidecode will fail to build.

Reply:
It didn't break anything -- https://jenkins-master-prod-1.delivery.puppetlabs.net/view/puppet-agent/view/ad-hoc/job/platform_puppet-agent-extra_puppet-agent-packaging_adhoc-ad_hoc/452/

Reply:
Context for those not super familliar with make vs. shell: This is changing the variable from being expanded by make to being expanded by the shell interpreter later.